### DISCUSSION TITLE
Is there a way to reorder rows using drag and drop?

### DISCUSSION THREAD

**Discussion by rohanrajpal** [bryanmylee/svelte-headless-table#146] [Q&A]
I have a use case in which we allow users to reorder the rows

![image](https://github.com/bryanmylee/svelte-headless-table/assets/7023147/e942aee3-3e86-4eff-80eb-6af9d376fed7)

since we've made our tables with this library, would be curious to know if there's a way to reorder the rows too!

**Comment by george-martinec** [comment]
Any news ?
